### DISCUSSION TITLE
Is it possible to include local css/js etc. ?

### DISCUSSION THREAD

**Discussion by paul-hansen** [tauri-apps/wry#597] [General]
When using local html files, is everyone just including it inline or is there a method to make local css and js files work? 

I tried using a custom protocol and it works for the initial html, but it doesn't work for including things in the html file.
``` html
<link rel="stylesheet" href="wry://localhost/static/style.css">
```
I get an `Failed to load resource: net::ERR_UNKNOWN_URL_SCHEME` error in the dev tools console when I load a page using that. I'm on Windows so it's using the Edge webview. Any ideas?

**Comment by amrbashir** [comment]
use absolute paths without wry scheme
```html
<link rel="stylesheet" href="/static/style.css">
```
then your custom protocol should parse the URL as it sees fit and send back the bytes, check `examples/custom_protocol.rs`

**Comment by paul-hansen** [comment]
Excellent, that works. Figured I was likely missing something obvious. Thank you!
